src/main/java/touro/peg/PlayMove.java

```diff
@@ -6,10 +6,46 @@
 public class PlayMove {
     private TriangleBoard board;
     private List<Move> movesList = new ArrayList<>();
-    private List<Move> legalMoves = new ArrayList<>();
+    private List<Move> legalMoves = List.of(
+            new Move(0,1,3),
+            new Move(3,1,0),
```

Comment:
This is not different than the way it was before. Every PlayMove object will have its own list of legal moves, rather than all sharing one.